docs/gender-aware-input.md

```diff
@@ -0,0 +1,271 @@
+<dd>
+    <i>File? </i><br />
+    The index for the bamfile given through bamFile.
+</dd>
+<dt id="GenderAwareVariantCalling.scatterAutosomalRegions.javaXmx"><a href="#GenderAwareVariantCalling.scatterAutosomalRegions.javaXmx">GenderAwareVariantCalling.scatterAutosomalRegions.javaXmx</a></dt>
+<dd>
+    <i>String </i><i>&mdash; Default:</i> <code>"8G"</code><br />
+    The maximum memory available to the program. Should be lower than `memory` to accommodate JVM overhead.
+</dd>
+<dt id="GenderAwareVariantCalling.scatterAutosomalRegions.memory"><a href="#GenderAwareVariantCalling.scatterAutosomalRegions.memory">GenderAwareVariantCalling.scatterAutosomalRegions.memory</a></dt>
+<dd>
+    <i>String </i><i>&mdash; Default:</i> <code>"24G"</code><br />
+    The amount of memory this job will use.
+</dd>
+<dt id="GenderAwareVariantCalling.scatterAutosomalRegions.notSplitContigs"><a href="#GenderAwareVariantCalling.scatterAutosomalRegions.notSplitContigs">GenderAwareVariantCalling.scatterAutosomalRegions.notSplitContigs</a></dt>
+<dd>
+    <i>Boolean </i><i>&mdash; Default:</i> <code>false</code><br />
+    Equivalent to biopet scatterregions' `--notSplitContigs` flag.
+</dd>
+<dt id="GenderAwareVariantCalling.scatterSize"><a href="#GenderAwareVariantCalling.scatterSize">GenderAwareVariantCalling.scatterSize</a></dt>
+<dd>
+    <i>Int </i><i>&mdash; Default:</i> <code>1000000000</code><br />
+    The size of the scattered regions in bases. Scattering is used to speed up certain processes. The genome will be sseperated into multiple chunks (scatters) which will be processed in their own job, allowing for parallel processing. Higher values will result in a lower number of jobs. The optimal value here will depend on the available resources.
+</dd>
+</dl>
+</details>
+
+
+
+## Other inputs
+<details>
+<summary> Show/Hide </summary>
+<dl>
+<dt id="GenderAwareVariantCalling.callX.excludeIntervalList"><a href="#GenderAwareVariantCalling.callX.excludeIntervalList">GenderAwareVariantCalling.callX.excludeIntervalList</a></dt>
+<dd>
+    <i>Array[File]+? </i><br />
+    ???
+</dd>
+<dt id="GenderAwareVariantCalling.callY.excludeIntervalList"><a href="#GenderAwareVariantCalling.callY.excludeIntervalList">GenderAwareVariantCalling.callY.excludeIntervalList</a></dt>
+<dd>
+    <i>Array[File]+? </i><br />
+    ???
+</dd>
+<dt id="GenderAwareVariantCalling.Gvcf.haplotypeCallerGvcf.excludeIntervalList"><a href="#GenderAwareVariantCalling.Gvcf.haplotypeCallerGvcf.excludeIntervalList">GenderAwareVariantCalling.Gvcf.haplotypeCallerGvcf.excludeIntervalList</a></dt>
+<dd>
+    <i>Array[File]+? </i><br />
+    ???
+</dd>
+<dt id="GenderAwareVariantCalling.Gvcf.haplotypeCallerGvcf.ploidy"><a href="#GenderAwareVariantCalling.Gvcf.haplotypeCallerGvcf.ploidy">GenderAwareVariantCalling.Gvcf.haplotypeCallerGvcf.ploidy</a></dt>
+<dd>
+    <i>Int? </i><br />
+    ???
+</dd>
+</dl>
+</details>
```

Comment:
These inputs seem to be missing a description and category.

Reply:
Yes I did not update haplotypecaller yet. I will  make an issue for that.